x-pack/test/functional/apps/saved_objects_management/multi_space_import.ts

```diff
@@ -0,0 +1,194 @@
+      await PageObjects.common.navigateToUrl('dashboard', undefined, {
+        basePath: getSpacePrefix(spaceId),
+        shouldUseHashForSubUrl: false,
+      });
+      await PageObjects.dashboard.loadSavedDashboard('multi_space_import_8.0.0_export');
+
+      // dashboard should load properly
+      await PageObjects.dashboard.expectOnDashboard('multi_space_import_8.0.0_export');
+      await PageObjects.dashboard.waitForRenderComplete();
+
+      // There should be 0 error embeddables on the dashboard
+      const errorEmbeddables = await testSubjects.findAll('embeddableStackError');
+      expect(errorEmbeddables.length).to.be(0);
+    });
+
+    it('should be able to copy the dashboard into third space in saved objects table using copy to space', async () => {
+      const destinationSpaceId = 'third_space';
+      const getSpacePrefix = (spaceId: string) => {
+        return spaceId && spaceId !== 'default' ? `/s/${spaceId}` : ``;
+      };
+      const spaceId = 'another_space';
+      await PageObjects.common.navigateToUrl('settings', 'kibana/objects', {
+        basePath: getSpacePrefix(spaceId),
+        shouldUseHashForSubUrl: false,
+      });
+      await PageObjects.savedObjects.waitTableIsLoaded();
+      await PageObjects.copySavedObjectsToSpace.openCopyToSpaceFlyoutForObject(
+        'multi_space_import_8.0.0_export'
+      );
+      await PageObjects.copySavedObjectsToSpace.setupForm({
+        createNewCopies: true,
+        overwrite: false,
+        destinationSpaceId,
+      });
+      await PageObjects.copySavedObjectsToSpace.startCopy();
+      // Wait for successful copy
+      await testSubjects.waitForDeleted(`cts-summary-indicator-loading-${destinationSpaceId}`);
+      await testSubjects.existOrFail(`cts-summary-indicator-success-${destinationSpaceId}`);
+
+      const summaryCounts = await PageObjects.copySavedObjectsToSpace.getSummaryCounts();
+
+      expect(summaryCounts).to.eql({
+        success: 6,
+        pending: 0,
+        skipped: 0,
+        errors: 0,
+      });
+      await PageObjects.copySavedObjectsToSpace.finishCopy();
+    });
+
+    it('should render all panels on the copied dashboard in third space', async () => {
+      const getSpacePrefix = (spaceId: string) => {
+        return spaceId && spaceId !== 'default' ? `/s/${spaceId}` : ``;
+      };
+      const spaceId = 'third_space';
+      await PageObjects.common.navigateToUrl('dashboard', undefined, {
+        basePath: getSpacePrefix(spaceId),
+        shouldUseHashForSubUrl: false,
+      });
+      await PageObjects.dashboard.loadSavedDashboard('multi_space_import_8.0.0_export');
+
+      // dashboard should load properly
+      await PageObjects.dashboard.expectOnDashboard('multi_space_import_8.0.0_export');
+      await PageObjects.dashboard.waitForRenderComplete();
+
+      // There should be 0 error embeddables on the dashboard
+      const errorEmbeddables = await testSubjects.findAll('embeddableStackError');
```

Comment:
I'm not feeling 100% confident this will catch all cases - could we check the number of `data-render-complete="true"` panels as well to make sure they rendered? In case an embeddable fails internally somehow (without switching to error embeddable) on broken references it won't be caught otherwise.